conans/client/conf/__init__.py

```diff
@@ -90,7 +100,8 @@
 compression_level = 9                 # environment CONAN_COMPRESSION_LEVEL
 sysrequires_sudo = True               # environment CONAN_SYSREQUIRES_SUDO
 request_timeout = 60                  # environment CONAN_REQUEST_TIMEOUT (seconds)
-# sysrequires_mode = enabled            # environment CONAN_SYSREQUIRES_MODE (allowed modes enabled/verify/disabled)
+plugins = attribute_checker           # environment CONAN_PLUGINS
```

Comment:
I think it will be more maintainable if a [plugins] section instead.

Reply:
Need to check what would be the way to "merge" plugins while installing them from ``conan config install``

Reply:
I wouldn't duplicate, if there is a dedicated [plugins] section, I wouldn't declare them also in another place.

Reply:
I like the idea of having a ``[plugins]`` section and that was my original thought but I am not sure what would be the syntax for ``conan set``: ``conan set plugins=my_plugin,my_other_plugin``?

Reply:
``conan set plugins.my_plugin``? The sections are supposed to be managed as ``section.field``